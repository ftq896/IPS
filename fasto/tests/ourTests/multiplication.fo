fun int main() =
  let res = write(2*3) in
  let res = write(6*~3) in
  let res = write(~5*~5) in
  write(~4*4)