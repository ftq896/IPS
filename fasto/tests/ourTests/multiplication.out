6
-18
25
-16